config/manifests/bases/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: kuadrant-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Kuadrant is the Schema for the kuadrants API
+      displayName: Kuadrant
+      kind: Kuadrant
+      name: kuadrants.kuadrant.kuadrant.io
+      version: v1beta1
+  description: Kuadrant Operator
+  displayName: Kuadrant
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
```

Comment:
:eyes:  

we have a operator package that does not deploy anything :) (step by step I know)

Reply:
That's just the kustomize base file, the generated CSV has a deployment https://github.com/Kuadrant/kuadrant-operator/pull/3/files#diff-2448a10ce1acbcd0da21db3f6da11959d0dc5897b2319efec8472b69dc5accb0R78

But this operator does do absolutely nothing yet, mainly want this now to add the CI builds.